server/core/store.go

```diff
@@ -425,13 +425,14 @@ func (s *StoreInfo) AvailableRatio() float64 {
 }
 
 // IsLowSpace checks if the store is lack of space. Not check if region count less
-// than initialMaxRegionCounts and available space more than initialMinSpace
+// than InitClusterRegionThreshold and available space more than initialMinSpace
 func (s *StoreInfo) IsLowSpace(lowSpaceRatio float64) bool {
 	if s.GetStoreStats() == nil {
 		return false
 	}
-	// issue #3444
-	if s.regionCount < initialMaxRegionCounts && s.GetAvailable() > initialMinSpace {
+	// See https://github.com/tikv/pd/issues/3444 and https://github.com/tikv/pd/issues/5391
+	// TODO: we need find a better way to get the init region number when starting a new cluster.
+	if s.regionCount < InitClusterRegionThreshold && s.GetAvailable() > initialMinSpace {
```

Comment:
maybe using the start time is better?

Reply:
Add a TODO here since using start time could also be inaccurate.